src/ProjectConfiguration.ts

```diff
@@ -623,6 +635,63 @@ export function validateCompanionDefaultWakeInterval(
   return diagnostics;
 }
 
+export function validateTileComponentAppType(config: ProjectConfiguration) {
+  const diagnostics = new DiagnosticList();
+
+  // Tiles are available just for appType = APP
+  // @ts-ignore
+  if (config.appType !== AppType.APP && config['tiles'] !== undefined) {
```

Comment:
Why the stringified key here instead of the normal `config.tiles`?

Reply:
No reason. Misspelled. Changing to normal approach.

Reply:
There are some legit cases where this applies, so just checking. Thanks for fixing! :)